lacework/alert_channel.go

```diff
@@ -0,0 +1,18 @@
+package lacework
+
+import (
+	"github.com/lacework/go-sdk/api"
+	"github.com/pkg/errors"
+)
+
+// VerifyAlertChannel tests the integration of an alert channel
+func VerifyAlertChannel(id string, lacework *api.Client) error {
+	if err := lacework.V2.AlertChannels.Test(id); err != nil {
+		// rollback terraform create upon error testing integration
+		if _, deleteErr := lacework.Integrations.Delete(id); deleteErr != nil {
```

Comment:
why not use v2 already here?
```suggestion
		if _, deleteErr := lacework.V2.AlertChannels.Delete(id); deleteErr != nil {
```